.github/workflows/llvm.yml

```diff
@@ -26,6 +26,8 @@ jobs:
             llvm_ubuntu_version: "22.04"
           - llvm_version: "17.0.6"
             llvm_ubuntu_version: "22.04"
+          - llvm_version: "18.1.4"
+            llvm_ubuntu_version: "18.04"
```

Comment:
**question:** Why Ubuntu 18.04?

Reply:
These release packages are built by volunteers and they don't necessarily have 22.04

Reply:
Why don't we use the packages from the [LLVM APT repositories](https://apt.llvm.org/)?